diff --git a/android/ExpoModulesCorePlugin.gradle b/android/ExpoModulesCorePlugin.gradle
index 1234567..abcdefg 100644
--- a/android/ExpoModulesCorePlugin.gradle
+++ b/android/ExpoModulesCorePlugin.gradle
@@ -85,22 +85,8 @@ ext.useDefaultAndroidSdkVersions = {
 }
 
 ext.useExpoPublishing = {
-  if (!project.plugins.hasPlugin('maven-publish')) {
-    apply plugin: 'maven-publish'
-  }
-
-  project.android {
-    publishing {
-      singleVariant("release") {
-        withSourcesJar()
-      }
-    }
-  }
-
   project.afterEvaluate {
     publishing {
       publications {
         release(MavenPublication) {
-          from components.release
         }
       }
       repositories {
